### PR TITLE
Add configurations for marketing consent banner

### PR DESCRIPTION
**Pull request by JayaShakthi97**
### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Review by thivi — APPROVED** [review]

File: apps/console/src/extensions/configs/application.tsx [hunk 1]
```diff
@@ -83,26 +78,35 @@ export const applicationConfig: ApplicationConfig = {
         getOverriddenImage: (_clientId: string, _tenantDomain: string) => {
             return null;
         },
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (brionmario) [inline apps/console/src/extensions/configs/application.tsx:81]: Remove suppression. Add `_` to the front of the var instead.
```suggestion
getTabPanelReadOnlyStatus: (_tabPanelName: string, _applicationName: ApplicationInterface)
```

File: apps/console/src/extensions/configs/application.tsx [hunk 2]
```diff
@@ -83,26 +78,35 @@ export const applicationConfig: ApplicationConfig = {
...
+        },
         isTabEnabledForApp: (_clientId: string, _tabType: ApplicationTabTypes, _tenantDomain: string): boolean => {
             return true;
         },
         renderHelpPanelItems: (): ReactNode => {
             return null;
         },
-        showProvisioningSettings: true,
+        showApplicationShare: true,
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (brionmario) [inline apps/console/src/extensions/configs/application.tsx:92]:
```suggestion
```

File: apps/console/src/extensions/configs/application.tsx [hunk 3]
```diff
@@ -83,26 +78,35 @@ export const applicationConfig: ApplicationConfig = {
...
         showDangerZone: (application: ApplicationInterface): boolean => {
             return true;
         },
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (brionmario) [inline apps/console/src/extensions/configs/application.tsx:96]:
```suggestion
```

File: apps/console/src/extensions/configs/application.tsx [hunk 4]
```diff
@@ -83,26 +78,35 @@ export const applicationConfig: ApplicationConfig = {
...
-        getTabPanelReadOnlyStatus: (tabPanelName: string, applicationName: ApplicationInterface): boolean => {
-            return false;
-        },
-        showApplicationShare: true
+        showProvisioningSettings: true
     },
     excludeIdentityClaims: false,
     excludeSubjectClaim: false,
+    generalSettings: {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (brionmario) [inline apps/console/src/extensions/configs/application.tsx:105]:
```suggestion
```

File: apps/console/src/extensions/configs/application.tsx (hunk 1 above)
Comment (JayaShakthi97) [reply apps/console/src/extensions/configs/application.tsx]: Fixed with b162ba9502b5fc4e15d504f3f3ae0ae36f94a755

File: apps/console/src/extensions/configs/application.tsx (hunk 2 above)
Comment (JayaShakthi97) [reply apps/console/src/extensions/configs/application.tsx]: Fixed with b162ba9502b5fc4e15d504f3f3ae0ae36f94a755

File: apps/console/src/extensions/configs/application.tsx (hunk 3 above)
Comment (JayaShakthi97) [reply apps/console/src/extensions/configs/application.tsx]: Fixed with b162ba9502b5fc4e15d504f3f3ae0ae36f94a755

File: apps/console/src/extensions/configs/application.tsx (hunk 4 above)
Comment (JayaShakthi97) [reply apps/console/src/extensions/configs/application.tsx]: Fixed with b162ba9502b5fc4e15d504f3f3ae0ae36f94a755

**Review by brionmario — APPROVED** [review]

**Pull request merged**